data/mm/world/overworld.yml

```diff
@@ -480,8 +522,8 @@
   exits:
     "Road to Ikana": "true"
     "Beneath The Graveyard Night 1": "has(MASK_CAPTAIN)"
-    "Beneath The Graveyard Night 2": "has(MASK_CAPTAIN)"
-    "Beneath The Graveyard Night 3": "has(MASK_CAPTAIN)"
+    "Beneath The Graveyard Night 2": "has(MASK_CAPTAIN) && can_use_lens"
+    "Beneath The Graveyard Night 3": "has(MASK_CAPTAIN) && has_weapon_range"
```

Comment:
You don't need anything more than the mask to enter the graves, these conditions should be on the checks (for future entrance rando)

Reply:
I modified the wrong line, this one is on me